http-api/src/lib.rs

```diff
@@ -82,7 +97,65 @@ impl Context {
         Ok(())
     }
 
-    /// Populate a map between ssh fingerprints and their peer identities
+    fn cleanup_sessions(&self, map: &mut HashMap<String, AuthState>) -> Result<(), Error> {
+        let mut to_remove: Vec<String> = Vec::new();
+        let current_time = self.now()?;
+
+        for (key, value) in map.iter() {
+            match value {
+                AuthState::Authorized(auth) => {
+                    if let Some(exp_time) = auth.expiration_time {
+                        if exp_time < current_time {
```

Comment:
Should be `>=` no?

Reply:
In this case we want to add expired sessions to a `to_remove` vector so we need to check if `exp_time` is smaller than the current timestamp.